src/libs/OptionsListUtils.js

```diff
@@ -133,7 +134,7 @@ function getPolicyExpenseReportOptions(report) {
  * @return {String}
  */
 function addSMSDomainIfPhoneNumber(login) {
-    if (Str.isValidPhone(login) && !Str.isValidEmail(login)) {
+    if (parsePhoneNumber(login).possible && !Str.isValidEmail(login)) {
         const smsLogin = login + CONST.SMS.DOMAIN;
         return smsLogin.includes('+') ? smsLogin : `+${countryCodeByIP}${smsLogin}`;
```

Comment:
Let's use `appendCountryCode` here.

Reply:
I'm not sure about that. Let's keep the footprint minimal and add the country code but use `appendCountryCode` instead.